io.github.gitahead.Gitahead.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.gitahead.Gitahead</id>
+  
+  <name>Gitahead</name>
+  <summary>Understand your Git history!</summary>
+  
+  <metadata_license>MIT</metadata_license>
```

Comment:
CC-0 is prefered for the metadata license.

Reply:
sure I can change it to that